CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## unreleased
 * Require Xcode 14.1 (per [App Store requirements](https://developer.apple.com/news/?id=jd9wcyov#:~:text=Starting%20April%2025%2C%202023%2C%20iOS,on%20the%20Mac%20App%20Store))
+* Send analytic events to FPTI instead of Arachne
```

Comment:
```suggestion
* Change analytics endpoint
```

Reply:
I don't think we need to publicly mention service names that we're changing to, what do you think?

Reply:
Good question - I was just thinking of what would be most clear for our future-selves when we want to hunt down when we made the switch.

Maybe we can be more generic like `Move to PayPal analytics service`?

Reply:
Yeah I think that’s a great compromise! 🚢

Reply:
Done!